ui/admin/app/controllers/scopes/scope/host-catalogs/host-catalog/host-sets/host-set/hosts/index.js

```diff
@@ -0,0 +1,43 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+
+export default class ScopesScopeHostsHostCatalogsHostCatalogHostSetsHostSetHostsController extends Controller {
+  // =services
+  @service intl;
+
+  // =attributes
+  /**
+   * Returns the columns for the host list.
+   * @type {Array.<object>}
+   */
+  get columns() {
+    if (this.model.hostSet.isStatic) {
+      return [
+        {
+          label: this.intl.t('form.name.label'),
```

Comment:
To improve readability - I'd move these properties to an array variable at the top and just return the var name here in the function.

Reply:
Indeed with @DhariniJeeva comment, good call Dharini!!

Reply:
@DhariniJeeva @ZedLi take a look at the changes and let me know what you think.